app/(auth)/index.tsx

```diff
@@ -0,0 +1,209 @@
+import { useRouter } from 'expo-router';
+import { useEffect, useRef, useState } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  Image,
+  TextInput,
+  KeyboardAvoidingView,
+  Platform,
+  TouchableOpacity,
+} from 'react-native';
+// import { LinearGradient } from 'expo-linear-gradient';
+import { useSession } from '@/context/AuthContext';
+import PokedexButtom from '@/components/PokedexBottom';
+import { signIn as reduxSignIn } from '@/redux/actions/pokemonAction';
+import { Dispatch } from '@reduxjs/toolkit';
+import { useDispatch } from 'react-redux';
+import { LED } from '@/components/ui/LED';
+export default function Login() {
+  const router = useRouter();
+  // const logo = require('@/assets/images/logo.png');
+  const [email, onChangeEmail] = useState('chuck.noris@gmail.com');
+  const [password, onChangePassword] = useState(
+    "Au commencement, il regarda Arceus crée l'univers",
+  );
+  // const [email, onChangeEmail] = useState('');
+  // const [password, onChangePassword] = useState('');
+  const [buttonDisabled, setButtonDisabled] = useState(true);
+  const { signIn } = useSession();
+
+  function checkEmail(email: string) {
+    let re =
+      /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    if (re.test(email)) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Pourquoi ne pas directement return la valeur ?

```suggestion
    return (re.test(email))
```